app/assets/stylesheets/v3/layout_components/forms.scss

```diff
@@ -178,14 +179,15 @@ input[type="checkbox"] + label,
     content: " ";
     display: block;
     position: absolute;
-    left: 10px;
-    top: calc(50% - 12px);
-    width: 24px;
-    min-width: 24px;
-    height: 24px;
+    left: 1rem;
+    top: calc(50% - 15px);
+    width: 30px;
```

Comment:
just curious why we need width and min-width both set?

Reply:
not sure, it was there before so i just left it. toggling it off in the browser doesn't seem to affect it though so I'll remove the min width.